packages/backend/src/apps/lettersg/auth/verify-api-key.ts

```diff
@@ -0,0 +1,9 @@
+import { IGlobalVariable } from '@plumber/types'
+
+export async function verifyApiKey($: IGlobalVariable): Promise<void> {
+  const response = await $.http.get('/v1/templates?limit=1')
+
+  if (!response?.data?.count) {
+    throw new Error('API key is invalid!')
+  }
```

Comment:
should we be checking for `HTTP 401 Unauthorized` instead?

Reply:
can consider changing the error message to "API key is invalid, please ensure you've copied the correct API key or contact support@plumber.gov.sg if you require assistance" 

Or

"API key is invalid, please ensure you've copied the correct API key"

Reply:
ok will do so, thanks!